### DISCUSSION TITLE
Additional views customization

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#447] [Ideas]
The current Tracks+Art view is perfect (left panel > playlist > art)! :heart: 

@Taiko2k  what do you think about these combinations? 

art > playlist > panel
playlist > panel > art
art > panel > playlist
panel > art > playlist

How difficult to implement this?

**Comment by Taiko2k** [comment]
Pretty hard. Technically its a bit of a nightmare but even from a design point of view, there's a lot of subtle visual balancing that's optimised for the current UI flow.

I can certainly see the appeal of more customisation. But I haven't managed to think of a way to do it that doesn't sacrifice the current level of polish, or involve an insurmountable amount of work.

**Reply by Solarunit** [reply]
I see

**Reply by Taiko2k** [reply]
On a similar note, I don't plan on adding any new features or making major changes soon. I plan on just focusing on bug fixes, stability and refinement for now.

**Reply by Solarunit** [reply]
Its already feature complete for me.

**Comment by StrangeSeasOfThought** [comment]
I'd love a combined (mini-)queue & arlist list view in the left panel, similiar to the combined playlist & queue panel. The size of the miniqueue perhaps 5 queued tracks.

**Reply by labatts** [reply]
and a way to keep the micro view above other windows.  But, perhaps that is a job for the window manager.  I'll look.

**Reply by akhilman** [reply]
Would it be possible to add a checkbox for queue in the left corner menu, allowing it to be combined with all other left panel modes, such as playlist, artist list, and folder navigator?

**Comment by Solarunit** [comment]
in v10 we can now put metadata panel (Art) on the left side! But if you use Playlist panel it would be still on the left side.

I wish we could place Playlist panel on the far right side.
